source/voting.rst

```diff
@@ -0,0 +1,96 @@
+.. _voting:
+
+############
+Voting Items
+############
+
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the project developers’ mailing list, the issue tracker, and in the
+project meetings mentioned above. Occasionally, sensitive discussion occurs on
+a private list, but the project defaults to open meetings.
+
+The yt-project uses a “consensus seeking” process for making large,
+project-changing decisions. The
+group tries to find a resolution that has no open objections among project
+members. At any point during the discussion, any project member can call for
+a vote, which will conclude two weeks from the call for the vote. Any vote must
+be backed by a YTEP. If no option can gather two thirds of the votes cast, the
+decision is escalated to the Steering Committee, which in turn will use
+consensus seeking with the fallback option of a simple majority vote if no
+consensus can be found within a month. 
+This is what we hereafter may refer to as “the decision making process”.
+
+Decisions (in addition to adding project members and Steering Committee
+membership as above) are made according to the following rules (+1 is
+equivalently an approval vote on a pull request):
+
+Minor Documentation Changes
+---------------------------
```

Comment:
I'm not sure I understand this section. I would expect any minor changes to happen in a PR and then the process for accepting PRs would apply.

Reply:
This is supposed to be describing the process for accepting PRs. Maybe that would be clear if I had +1/accept and -1/reject in the language body? I've removed the thing about issues in this subsection because I think we don't need to approve issues for minor changes.

Reply:
Could you make the text below a little clearer? I understand what you mean and agree that it should work that way after reading your comment but I agree with Britton it could be more explicit and clearer in the text.

Reply:
Done.